instrumentation/kamon-elasticsearch/src/main/java/kamon/instrumentation/elasticsearch/AsyncElasticsearchRestClientInstrumentation.java

```diff
@@ -15,7 +15,7 @@ public static void enter(
         final String operationName =
                 RequestNameConverter.convert(
                         HighLevelElasticsearchClientInstrumentation.requestClassName.get(),
-                        "elasticsearch.async");
+                        "AsyncRequest");
```

Comment:
This one should be `elasticsearch/AsyncRequest`, same goes for `elasticsearch/SyncRequest`.

Reply:
"AsyncRequest" and "SyncRequest" are fallbacks for classnames, like "ListTasksRequest" (which might be null).
So they resulting operation name is "elasticsearch/ClassName".